core/src/types.ts

```diff
@@ -266,3 +266,51 @@ export declare interface DebtAuctionEnriched extends DebtAuctionActive {
 }
 
 export declare interface DebtAuctionTransaction extends DebtAuctionEnriched, CompensationAuctionTransactionFees {}
+
+type CollateralType = CollateralConfig['title'];
+export declare interface VaultCollateralType {
+    collateralType: CollateralType;
+}
+
+export declare interface LiquidationLimit {
+    maximumAmountAuctionedCollateral: BigNumber;
+    currentlyAuctionedCollateral: BigNumber;
+}
+
+export declare interface VaultCollateralParameters extends LiquidationLimit, VaultCollateralType {
+    stabilityFeeRate: BigNumber;
+    maxDebtPerCollateralUnit: BigNumber;
+}
+
+export declare interface VaultBase extends VaultCollateralType {
+    id: number;
+    address: string;
+}
+
+export declare interface VaultAmount {
+    initialDebtDai: BigNumber;
+    collateralAmount: BigNumber;
+    collateralType: CollateralType;
+}
+
+export declare interface VaultTransactionFees {
+    transactionFeeLiquidationEth: BigNumber;
+    transactionFeeLiquidationDai: BigNumber;
+}
+
+export declare interface VaultBaseInfo extends VaultBase, VaultAmount {}
+export declare interface OraclePrices {
+    current: BigNumber;
+    next: BigNumber;
```

Comment:
If `OraclePrices` are used to expend into `Vault`, those keys should have better names
```suggestion
    currentUnitPrice: BigNumber;
    nextUnitPrice: BigNumber;
```